jetson/odrive_bridge/src/__main__.py

```diff
@@ -258,28 +251,25 @@ def on_event(self, event):
         publish_state_msg(state_msg, odrive_bridge.get_state())
 
     def update(self):
-        try:
-            errors = modrive.check_errors()
-            modrive.watchdog_feed()
-        except Exception:
-            errors = 0
-            lock.acquire()
-            self.on_event("disconnected odrive")
-            lock.release()
-            print("unable to check errors of unplugged odrive")
+        if (str(self.state) == "ArmedState"):

```

Comment:
Using enums would clean up comparisons like this

Reply:
state is a class variable, not exactly sure what you mean

Reply:
I was thinking that if you had made the events and state variables enums, then you wouldn't need to be casting your state to a string and could just compare it with an enum.

Reply:
i don't cast it, state is a class and str is like a function/property of the class, so for each state i would need an extra object in the class that's an enum defining it, such as state.NAME that returns that enum, which seems like pretty much the same thing in my opinion. i can change the event names to be enums although that doesn't do anything to the code clarity imo

Reply:
Yeah, I didn't realize it was a class. That's my bad. Please change the event names to enums, since I do think that would be better coding practice

Reply:
sure